study-builder/studies/basil/study-activities.conf

```diff
@@ -20,7 +20,29 @@
     },
     {
       "filepath": "loved-one-activity.conf",
-      "mappings": []
+      "mappings": [],
+      "validations": [
+        {
+          # test for ERROR_PRESENT status.
+          "stableIds": ["ERROR_PRESENT_COMMENTS"],
+          "allowSave": true,
+          "precondition": ${_pex.is_comments_answered},
+          "expression": ${_pex.is_comments_error},
+          "messageTemplate": {
+            "templateType": "HTML",
+            "templateText": "$error_present_comm",
+            "variables": [
+              {
+                "name": "error_present_comm",
+                "translations": [
+                  { "language": "en", "text": ${i18n.en.aboutLovedOne.summary_error_present} },
```

Comment:
Shouldn't we use new translations mechanism?

Reply:
fon is not using new translations, so doesn't matter